cmd/lambda/providercache/main.go

```diff
@@ -25,6 +25,11 @@ func main() {
 	lambda.Start(makeHandler)
 }
 
+type handlerResult struct {
+	id  string
+	err error
+}
```

Comment:
no need to change it, just commenting in case you are curious: it is common practice to declare these types that are scoped to a function within the function itself, right before creating the channel

Reply:
FML